### PR TITLE
Testing documentation workflow

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
Going to deploy and then test to see if workflow works.

### CODE REVIEW

**Comment by kevindougherty-noaa** [comment]
@CoryMartin-NOAA I think I have exhausted everything I can think of to get this documentation to deploy. I have the fork of this repo pointed to deploy on this branch, added .nojekyll files, and everything is passing..

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -45,15 +45,15 @@ jobs:
...
-        cd $GITHUB_WORKSPACE
-        pdoc --docformat "google" -o docs emcpy
-        ls -lR docs/
-
-    - name: Upload documentation
-      uses: actions/upload-artifact@v2
-      with:
-        name: docs
-        path: docs
+#     # Generate documentation
```
Comment (CoryMartin-NOAA) [inline .github/workflows/tests.yaml]: why comment out, and not remove?

File: docs/conf.py [hunk 2]
```diff
@@ -16,11 +16,11 @@
 from datetime import datetime
 sys.path.insert(0, str(Path(__file__).parent.resolve()))
 
-import matplotlib
+# import matplotlib
 from sphinx_gallery.sorting import ExampleTitleSortKey, ExplicitOrder
 
 # sys.path.append(os.path.abspath('../src'))
-import emcpy
+# import emcpy
```
Comment (CoryMartin-NOAA) [inline docs/conf.py]: why comment these out?

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (kevindougherty-noaa) [reply .github/workflows/tests.yaml]: I was just testing

File: docs/conf.py (hunk 2 above)
Comment (kevindougherty-noaa) [reply docs/conf.py]: I was getting errors and was testing.

**Pull request merged**